app/controllers/books_controller.rb

```diff
@@ -2,8 +2,46 @@ class BooksController < ApplicationController
   def index
     @books = Book.all
   end
-  
+
   def show
     @book = Book.find(params[:id])
   end
+
+  def new
+    @book = Book.new
+  end
+
+  def create
+    whatever = author_params[:name].split(", ")
+    multiple_authors = []
+    whatever.each do |author|
+      new_author = Author.new(name: author)
+      if new_author.save
+        multiple_authors << new_author
+      else
+        multiple_authors << Author.find_by(name: author_params[:name])
```

Comment:
You might want to do this lookup before trying to make/save a new author, unless you have a uniqueness validation on the author name.